docs/faq.md

```diff
@@ -111,6 +111,16 @@ Also Shorebird does not support changing native code (e.g. Java/Kotlin on
 Android or Objective-C/Swift on iOS). The tool will warn you during an
 attempted patch if you have changed native code.
 
+### Can I use Shorebird for all my Dart changes?
+
+Technically speaking, yes, a Shorebird patch can update any Dart code.
+Note that depending on how you distribute your apps, some store agreements
```

Comment:
```suggestion
However, some store agreements
```

Reply:
I think leaving it as is is more explicit personally 🤷